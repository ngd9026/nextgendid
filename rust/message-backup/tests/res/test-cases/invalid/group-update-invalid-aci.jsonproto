[
  {
    "version": "1",
    "backupTimeMs": "1705692409729",
    "mediaRootBackupKey": "q6urq6urq6urq6urq6urq6urq6urq6urq6urq6urq6s=",
  },
  {
    "recipient": {
      "id": "1",
      "self": {}
    }
  },
  {
    "recipient": {
      "id": "2",
      "group": {
        "masterKey": "YWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWE=",
        "snapshot": {}
      }
    }
  },
  {
    "chat": {
      "id": "1",
      "recipientId": "2"
    }
  },
  {
    "chatItem": {
      "chatId": "1",
      "authorId": "1",
      "dateSent": 12345,
      "directionless": {},
      "updateMessage": {
        "groupChange": {
          "updates": [
            {
              "selfInvitedToGroupUpdate": {
                "inviterAci": "YmJiYmJiYmJiYmJiYmIK"
              }
            }
          ]
        }
      }
    }
  }
]