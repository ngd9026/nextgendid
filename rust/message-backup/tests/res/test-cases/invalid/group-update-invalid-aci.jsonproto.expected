chat frame ChatId(1) error: chat item 12345: group update: group update: SelfInvitedToGroupUpdate.inviterAci: invalid ACI